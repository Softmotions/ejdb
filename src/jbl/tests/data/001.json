{
  "str": "\uD834\uDF06",
  "str1": "15\u00f8C 3\u0111",
  "str2": "\u004d\u0430\u4e8c\ud800\udf02",
  "str3": "привет",
  "na": [
    0.00011999999999999999,
    6E-06,
    1E-06
  ],
  "glossary": {
    "title": "example glossary",
    "GlossDiv": {
      "title": "S",
      "GlossList": {
        "GlossEntry": {
          "ID": "SGML",
          "SortAs": "SGML",
          "GlossTerm": "Standard Generalized Markup Language",
          "Acronym": "SGML",
          "Abbrev": "ISO 8879:1986",
          "GlossDef": {
            "para": "A meta-markup language, used to create markup languages such as DocBook.",
            "GlossSeeAlso": [
              "GML",
              "XML",
              true,
              false
            ]
          },
          "GlossSee": "markup"
        }
      }
    }
  }
}