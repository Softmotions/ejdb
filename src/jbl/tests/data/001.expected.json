{
 "str": "𝌆",
 "str1": "15øC 3đ",
 "str2": "Mа二𐌂",
 "str3": "привет",
 "na": [
  0.1,
  0.000006,
  0.000001
 ],
 "glossary": {
  "title": "example glossary",
  "GlossDiv": {
   "title": "S",
   "GlossList": {
    "GlossEntry": {
     "ID": "SGML",
     "SortAs": "SGML",
     "GlossTerm": "Standard Generalized Markup Language",
     "Acronym": "SGML",
     "Abbrev": "ISO 8879:1986",
     "GlossDef": {
      "para": "A meta-markup language, used to create markup languages such as DocBook.",
      "GlossSeeAlso": [
       "GML",
       "XML",
       true,
       false
      ]
     },
     "GlossSee": "markup"
    }
   }
  }
 }
}